app/src/main/java/com/example/genericapplication/MainActivity2.kt

```diff
@@ -71,63 +77,88 @@ class MainActivity2 : AppCompatActivity() {
         this.webView.settings.allowContentAccess = true
     }
 
+    private fun loadWebViewClient() {
+        webView.webViewClient = object : WebViewClient() {
+            override fun shouldOverrideUrlLoading(view: WebView, url: String): Boolean {
+                var operation = url.split("/")[1]
+                if (operation.equals("play")) {
+                    onResume()
+                }
+                if (operation.equals("pause")){
+                    onPause()
+                }
+                if (operation.equals("skip")){
+                    skipAds()
+                }
+                if (operation.equals("new")){
+                    videoUrl = getMediaUrl(url.split("/")[2])
+                    releasePlayer()
+                    initializePlayer()
+                }
+                return true
+            }
+        }
+    }
+
     private fun releasePlayer() {
         adsLoader!!.setPlayer(null)
         playerView!!.player = null
-        if(player != null)
-            player!!.release()
+        player!!.release()
```

Comment:
@DiegoA-coder I had an error here when the player is not required on the `MainActivity2`, this change is correct? The error occurs me when I return from Basic webview to dashboard.